graphene/src/vec2.rs

```diff
@@ -1,6 +1,6 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-use std::fmt;
+use std::{fmt, ops};
```

Comment:
No need to import std::ops here, you didnt do so in the other file

Reply:
I wanted to maintain consistency with importing `std::fmt` for `fmt::Display`. I've fixed `Matrix` to implement that.